contracts/DCAStrategies/DCAStrategies/DCAStrategiesManagementHandler.sol

```diff
@@ -14,10 +14,16 @@ abstract contract DCAStrategiesManagementHandler is IDCAStrategiesManagementHand
   /// @inheritdoc IDCAStrategiesManagementHandler
   uint80 public strategyCounter;
   uint16 internal constant _TOTAL = 100e2;
+  uint8 internal immutable _maxTokenShares;
```

Comment:
Let's make this public, so that we can read it later. Also, we use the same styling for immutables as constants, so it would look like `MAX_TOKEN_SHARES`

Finally, let's add some docs for it